build.rs

```diff
@@ -129,6 +129,35 @@ fn build_v8() {
     gn_args.push("host_cpu=\"arm64\"".to_string())
   }
 
+  let target = env::var("TARGET").unwrap();
+  if cfg!(target_os = "macos") && target.contains("apple-ios") {
+    // Gn configurations for iOS: https://v8.dev/docs/cross-compile-ios
+    let arch = {
+      if target.starts_with("aarch64") {
+        "amd64"
+      } else {
+        "x64"
+      }
+    };
+
+    let ios_deployment_target =
+      env::var("IOS_DEPLOYMENT_TARGET").unwrap_or("10".to_string());
```

Comment:
iOS 10 is old enough because iOS 16 is expected to release this year.

Not sure whether we need `IOS_DEPLOYMENT_TARGET` for custimization...